namerd/storage/k8s/src/main/resources/META-INF/services/io.buoyant.namerd.DtabStoreInitializer

```diff
@@ -1 +1 @@
-io.buoyant.namerd.storage.K8sDtabStoreInitializer
```

Comment:
looks like this removed the newline from the end of the file.  please revert this.